README.md

```diff
@@ -35,27 +35,24 @@ Atlas will serve as a bridge from existing bare-metal HPC clusters (example: Sat
 ## 4. Solution Concept
 ### Global Architectural Structure Of the Project:
 Below is a description of the system components that are building blocks of the architectural design:
-- **Scripts/Executables:** Users can write the scripts/executables to upload their codes and metadata of their program.
-- **Tasks Services:** Tasks Services will run in containers; once a user finishes uploading code and metadata, the tasks monitor service will be triggered to generate a task service in a container.
-- **Tasks monitor service:** This service should also run in a container, and monitor the execution status of a task, once a task finishes or fails, this service should report it. And it should also monitor all the key phases of a task, such as data preprocessing, data training, data prediction and so on, it can obtain the information from the metadata of this task which is uploaded with codes. After tasks finishing, this service should gather all the task results and organize them to form a comparison result between the cloud-native and bare-metal system.
-- **Data source:** The training data, validation data, test data and any other chunk data with relation to the task should be stored outside the system, they can be on AWS or individual machines.
+- **Scripts/Executables:** Users can write the scripts/executables to specify the commands for OpenShift and Satori.
+- **Containers:** The containers include the codes of AI program. And deployed by OpenShift
+- **Volumes:** Used for save training/test/validation data of AI program as well as results of the program.
```

Comment:
We might want to consider separating read-only volumes and write volumes? Since we might run multiple times of the program, we probably don't want to write results in the exact same place?

Reply:
Sure, we can do it later.